.env

```diff
@@ -96,6 +96,7 @@ _APP_VCS_GITHUB_APP_ID=
 _APP_VCS_GITHUB_CLIENT_ID=
 _APP_VCS_GITHUB_CLIENT_SECRET=
 _APP_VCS_GITHUB_WEBHOOK_SECRET=
+_APP_VCS_TEST_GITHUB_INSTALLATION_ID=
```

Comment:
Let's confirm variable name with Christy. I would probably suggest keeping it away from _APP scope. Or making TEXT primary. Soo something like

- `_APP_TESTS_VCS_GITHUB_INSTALLATION_ID`
- `_TESTS_VCS_GITHUB_INSTALLATION_ID`